### PR TITLE
adding tests for snack

### PR DESCRIPTION
**Pull request by atiyyaj2212**

### CODE REVIEW

File: src/test/java/jerry/codecamp/DogTest.java [hunk 1]
```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
...
+        assertEquals(10, dog.getThirstLevel());
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesBoredLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(-5, dog.getBoredLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-10, dog.getBoredLevel());
```
Comment (midnjerry) [inline src/test/java/jerry/codecamp/DogTest.java]: Create a setter OR do something that makes the boredom go higher before trying to go down.

File: src/test/java/jerry/codecamp/DogTest.java [hunk 2]
```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
...
+
+        dog.giveTheDogASnack();
+        assertEquals(10, dog.getDirtyLevel());
+
+        // hungerLevel -= 10;
+        //        lonelyLevel -= 10;
+        //        dirtyLevel += 5;
+        //        thirstLevel += 5;
+        //        boredLevel -= 5;
+
```
Comment (midnjerry) [inline src/test/java/jerry/codecamp/DogTest.java]: Please remove this.

File: src/test/java/jerry/codecamp/DogTest.java [hunk 3]
```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
...
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesLonelyLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(-10, dog.getLonelyLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getLonelyLevel());
+    }
```
Comment (midnjerry) [inline src/test/java/jerry/codecamp/DogTest.java]: Giving a snack increases or decreases?

Change to decreases.

Either do something that makes the LonelyLevel Go UP or make a setter and set the value to something > 20.

File: src/test/java/jerry/codecamp/DogTest.java [hunk 4]
```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
...
 
+    @Test
+    public void giveTheDogASnack_increasesHungerLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(- 10, dog.getHungerLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getHungerLevel());
+
+    }
```
Comment (midnjerry) [inline src/test/java/jerry/codecamp/DogTest.java]: Can a dog level go below 0?

**Review by midnjerry — COMMENTED** [review]
HI Atiyya,

Can you please make these changes.

File: src/test/java/jerry/codecamp/DogTest.java (hunk 1 above)
Comment (atiyyaj2212) [reply src/test/java/jerry/codecamp/DogTest.java]: do you mean dont go below zero?

Comment (midnjerry) [reply src/test/java/jerry/codecamp/DogTest.java]: Oh yes.  don't go below 0.

File: src/test/java/jerry/codecamp/DogTest.java (hunk 4 above)
Comment (atiyyaj2212) [reply src/test/java/jerry/codecamp/DogTest.java]: I notice I should the method name to decreases as well

**Review by atiyyaj2212 — COMMENTED** [review]
Can you review the changes

**Review by midnjerry — APPROVED** [review]
Great job!

**Pull request merged**